### DISCUSSION TITLE
An Potential Error in source code ComputeLinearElasticPFFractureStress.C

### DISCUSSION THREAD

**Discussion by DanielChou0916** [idaholab/moose#27663] [Q&A Modules: Solid mechanics]
Dear MOOSE community
I'm currently doing research on phase field fracture model, and when I take a look as the source code "ComputeLinearElasticPFFractureStress.C"
I found that in "computeStrainSpectral" function, there is an potential error.
In the source file:
https://github.com/idaholab/moose/blob/next/modules/solid_mechanics/src/materials/ComputeLinearElasticPFFractureStress.C
Line 106 is given as
`F_neg = -lambda * etrneg * etrneg / 2.0 + mu * nval;`
However, the negative sign on the first term ` -lambda * etrneg * etrneg / 2.0` does not make sense to me.
In the official website, the compressive part of elastic energy is given as shwon in following figure:
https://mooseframework.inl.gov/source/materials/ComputeLinearElasticPFFractureStress.html
![image](https://github.com/idaholab/moose/assets/142940612/044526c7-7c1c-4b5a-b290-40e6c916c7b7)
I also checked several papers to verify this portion, such as
https://www.sciencedirect.com/science/article/pii/S0020768324001744?ref=cra_js_challenge&fr=RR-1 (page 3, equation 11)
And 
https://materialstheory.springeropen.com/articles/10.1186/s41313-021-00037-1 (page 5, equation 16).

Could someone verify if this negative sign is indeed an error? If so, correcting it might be important for ensuring the accuracy of the energy calculations in the phase field fracture models.

Appreciate for your time and patience.
Sincerely

**Comment by GiudGiud** [comment]
@jiangwen84
